metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -862,6 +863,35 @@ class Compilers(
     }
   }.getOrElse(Future.successful(Nil.asJava))
 
+  def insertInferredMethod(
+      params: TextDocumentPositionParams,
+      token: CancelToken,
+  ): Future[WorkspaceEdit] = {
+    withPCAndAdjustLsp(params) { (pc, pos, adjust) =>
+      pc.insertInferredMethod(CompilerOffsetParamsUtils.fromPos(pos, token))
+        .asScala
+        .map(workspaceEdit =>
+          new WorkspaceEdit(
+            workspaceEdit
+              .getChanges()
+              .asScala
+              .map {
+                case (uri, textEdits) => {
+                  (
+                    uri,
```

Comment:
Maybe the uri here is wrong? Let's only return TextEdits maybe and use the uri from params like in ExtractMethodCodeAction?

Reply:
the url here is correct, I found out it is an issue happening on restart of the server only (that is the way I test it).